CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 ### Fixed
 - Exceptions occurring during async request now generate rejected promise (as they should) and are no longer thrown directly.
 
+### Added
+- Support for setting `properties` parameter in `UserRecommendation`.
```

Comment:
This is written more from API point of view; what about something like
`- Ability to set properties expected in recommendation response (via add...() or set...())`